flowcraft/generator/components/assembly.py

```diff
@@ -4,6 +4,50 @@
 except ImportError:
     from flowcraft.generator.process import Process
 
+class Bcalm(Process):
+    """Bcalm process template interface
+
+    This process is set with:
+
+        - ``input_type``: fastq
+        - ``output_type``: assembly
+        - ``ptype``: assembly
+
+    """
+
+    def __init__(self, **kwargs):
+
+        super().__init__(**kwargs)
+
+        self.input_type = "fastq"
+        self.output_type = "fasta"
+
+        self.dependencies = ["integrity_coverage"]
```

Comment:
From what I see in the nextflow template, this dependency is not really required, right?